superset/forms.py

```diff
@@ -343,6 +343,13 @@ def __init__(self, viz):
                     "Bottom marging, in pixels, allowing for more room for "
                     "axis labels"),
             }),
+            'page_length': (FreeFormSelectField, {
+                "label": _("Page Length"),
+                "default": 0,
+                "choices": self.choicify([0, 10, 25, 50 , 100, 250, 500]),
```

Comment:
That list of choices doesn't really matter much since it's a `FreeFormSelectField` anyways. The previous behavior is the equivalent of `0` which is all rows after the row_limit is applied. Though usually with pagination you're trying to fit things in the viewport, so anything of 100 probably can't fit though with large high res screen nowadays...